assets/xml/overlays/ovl_En_Clear_Tag.xml

```diff
@@ -0,0 +1,14 @@
+<Root>
+    <File Name="ovl_En_Clear_Tag" BaseAddress="0x809D35B0" RangeStart="0x26F0" RangeEnd="0x89F0" Segment="128">
+        <DList Name="arwingDL" Offset="0x26F0"/>
+        <DList Name="laserDL" Offset="0x6388"/>
+        <DList Name="arwingBackfireDL" Offset="0x6688"/>
```

Comment:
I think zel noted already, but these need to follow our naming convention.
`gArwing<NameOfThing><Suffix>`
siffixes are DL, Anim, Col etc
can use ArwingEff for the effects maybe

Reply:
Ah I missed the g part. Thanks!